src/views/imaged-object-editor/imaged-object-editor.vue

```diff
@@ -368,14 +375,20 @@ export default class ImagedObjectEditor
         (this.$refs.newArtefactName as any).focus();
     }
 
-   // moved code to created()
-   // from  private async mounted() {
-   // in order to have the masterImage ready before mounted
+
+   // moved code to created() from mounted() {
+   // in order to have the masterImage and other items
+   // (i.e this.artefactId ) ready before mounted
    // for the <zoomed...><svg> ... part
+   // and prevent errors of undefined parts
+   // occuring during render befor mounted,
+   // e.g. selectedRow: art.id === artefact.id returns undefined
+
+   private async created() {
 
-    private async created() {
         try {
```

Comment:
It probably would be good to have a corresponding `catch`.  But maybe you handle this possibility elsewhere.

Reply:
You're right, but I'd rather leave this for now. We will add a catch that will pop the corrupted state message, but for now I'm not doing this - all those errors in the console point us to bugs.